spec/features/audit/allocation/allocate_spec.rb

```diff
@@ -68,6 +68,29 @@
       expect(page).to_not have_content("Sense and Sensibility")
     end
 
+    scenario "Allocate content within current page and apply filters" do
```

Comment:
Could you please be a bit more descriptive about what it is you're doing here? Something like "My content tab shows number of items allocated to me regardless of page filters"?

Reply:
👍